scripts/pylib/twister/twisterlib/environment.py

```diff
@@ -270,6 +270,17 @@ def add_parse_arguments(parser = None) -> argparse.ArgumentParser:
         """
     )
 
+    parser.add_argument(
+        "--west-cmd", choices=['flash', 'debug'],
+        default='flash',
```

Comment:
you can remove the default here,
in handlers.py:594:
`west_cmd = self.options.west_cmd or hardware.west_cmd or "flash"`
in harness.py:483:
```
        if west_cmd := hardware.west_cmd or options.west_cmd:
            command.append(f'--west-cmd={west_cmd}')
```